### ISSUE TITLE
Accessibility: MAS 42A: High Contrast: In high contrast white theme the dropdown button for "Download for Windows" is missing 

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#11]
#A11yMAS
#MAS42A 
#Wipro 
#vscode-website
#Firefox 

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Prerequisite:**
**_1. Change system setting to High contrast White_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast White" from the "Choose a theme" dropdown. Click on Apply.

**Repro Steps:**
1.Open https://code.visualstudio.com/ in Edge.
2.Navigate to the "Download for Windows"  below the "Code editing. Redefined"  in top of the page.
3.Observe the  dropdown button for the "Download for Windows"

**Actual Result:**
The "Dropdown" button beside  "Download for Windows" is not visible in high contrast white  theme.

**Expected Result:**
The "Dropdown" button beside  "Download for Windows" should be visible in high contrast white theme.

**User Impact:**
The persons with color blindness and low vision will not be able to understand that there is a dropdown button for the download and may not use the application efficiently.

[Dropdown button.zip](https://github.com/wiproaccess/vscode-website/files/686947/Dropdown.button.zip)
![in black background also same issue firefox](https://cloud.githubusercontent.com/assets/24825441/21676582/173d0714-d35c-11e6-8039-2eb6dc12c745.png)

**Comment by waderyan** [comment]
Fixed with 2b4e904688206594e2c4cbcb33974566b74459ff
